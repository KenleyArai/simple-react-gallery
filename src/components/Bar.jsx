import styled from "styled-components";

const Bar = styled.div`
  width: 100vw;
  height: 5rem;
`;

export default Bar;
