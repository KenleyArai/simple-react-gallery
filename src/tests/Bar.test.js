import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Bar from "components/Bar";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Bar", () => {
  it("should render bar", () => {
    const wrapper = shallow(<Bar />);
  });
});
