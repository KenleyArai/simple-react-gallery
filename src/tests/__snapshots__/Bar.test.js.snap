// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar should render bar 1`] = `
<div
  className=""
/>
`;
